cpp/src/Slice/FileTracker.cpp

```diff
@@ -9,12 +9,6 @@
 using namespace IceInternal;
 using namespace std;
 
-const char*
-Slice::FileException::ice_id() const noexcept
-{
-    return "::Slice::FileException";
-}
-
```

Comment:
This exception is only used internally by the Slice compilers.
We never check or care about it's id. So why bother defining it.

Reply:
Whenever we define a LocalException, it needs to be a proper LocalException, with an ice_id(). We should never "half-define" a LocalException.

Reply:
It is now a `runtime_error` instead of a `LocalException`.

It really has no business being a `LocalException` since it doesn't interact at all with the Ice runtime.
And none of the functionality offered by `LocalExecption` was ever used by it.